spec/kickstart/main.fmf

```diff
@@ -0,0 +1,71 @@
+story:
```

Comment:
@psss is there a way how to express this is not implemented yet? The example below is more or less correct, but it's not implemented in Artemis, and therefore it will not work when this PR gets merged.

Reply:
Yes, stories which do not have a link with the `implemented-by` relation are shown with status `idea` and a note at the top of the story, here's an example:

* https://tmt.readthedocs.io/en/latest/spec/hardware.html#network

Is that sufficient?

Reply:
I believe so, yes. Thank you.